src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+
+const sortSex = (gender) => {
+  if (gender === 'm') {
+    return 'wife';
+  }
+
+  return 'husband';
+};
+
+export const Person = ({ person }) => (
+  <div className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    <div
+      className="Person__age"
+      style={{
+        display: person.age <= 0 ? 'none' : undefined,
+      }}
+    >
+      {person.age > 0
+
+        ? (`I am ${person.age}`)
+
+        : undefined}
+    </div>
```

Comment:
instead of using styles you can just not render an element
```suggestion
      {age && (
        <p className="Person__age">
          {`I am ${age}`}
        </p>
      )}
```